prime-router/src/main/kotlin/ConsolidatedActionLog.kt

```diff
@@ -0,0 +1,78 @@
+package gov.cdc.prime.router
```

Comment:
Thanks for adding documentation!!!  I added some requests here, but I am unsure what is new and what is old.  For old stuff that you have not changed feel free to skip any requests for kdoc.  For stuff you changed then might as well fix it.  Thx.